<launch>
    <node name="arm_se_arm" pkg="arm_se" type="arm_control" />
    <node name="arm_se_voice" pkg="arm_se" type="voicecontroller.py" />
    <node name="arm_se_slam" pkg="arm_se" type="navigation_multi_goals" />
    <node name="arm_se_control_center" pkg="arm_se" type="control_center" output="screen"/>
<!--     <node name="arm_se_camera" pkg="d435_camera" type="camera" /> -->
</launch>